pkg/v3/observation.go

```diff
@@ -59,17 +59,26 @@ func DecodeAutomationObservation(data []byte) (AutomationObservation, error) {
 	}
 
 	metadata := make(map[ObservationMetadataKey]interface{})
-	for key, value := range rawObs.Metadata {
+	for key := range rawObs.Metadata {
 		switch ObservationMetadataKey(key) {
 		case BlockHistoryObservationKey:
 			// value is a block history type
-			var bh ocr2keepers.BlockHistory
-
-			if err := json.Unmarshal(value, &bh); err != nil {
-				return obs, err
-			}
-
-			metadata[BlockHistoryObservationKey] = bh
+			// var tmp string
+			// var bh ocr2keepers.BlockKey
+
```

Comment:
Note: commenting in this PR as this breaks with new types. Not doing a proper fix as the overall logic needs to be rethought